builders/build/historical-updates.md

```diff
@@ -62,6 +62,29 @@ For more information, you can review the [relative PR on GitHub](https://github.
 
 ***
 
+### New Account Event Bug {: #new-account-event }
```

Comment:
Should it be #### instead of ###?

Reply:
yes! good catch! fixed this in a separate PR 🙂